refinery_core/src/traits/sync.rs

```diff
@@ -64,61 +30,76 @@ fn migrate_grouped<T: Transaction>(
     target: Target,
     migration_table_name: &str,
 ) -> Result<Report, Error> {
-    let mut grouped_migrations = Vec::new();
+    migrate_batch(transaction, migrations, target, migration_table_name, true)
+}
+
+fn migrate_batch<T: Transaction>(
```

Comment:
thanks for this cleanup! But can we can keep only `migrate` instead, i.e. rename this to `migrate` and the remove the `migrate` function that calls this one

Reply:
sure, and if I do that I think it makes sense to get rid of `migrate_grouped` too